tests/acceptance/pageObjects/sharedWithMePage.js

```diff
@@ -13,6 +13,31 @@ module.exports = {
       return navigationHelper.navigateAndWaitTillLoaded(
         this.url(), this.page.FilesPageElement.filesList().elements.filesListProgressBar.selector
       )
+    },
+    declinedStatusForDeletedFile: function (filename) {
```

Comment:
This is a kind of `assert` statement, right (i.e. asserting declined status is being shown for the specific file). Please name this accordingly.